app/views/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<%
+  items ||= []
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+  input_name ||= "reorderable-list-#{SecureRandom.hex(4)}"
+%>
+
+<%= tag.ol class: "app-c-reorderable-list", data: data_attributes do %>
+  <% items.each do |item| %>
+    <%= tag.li class: "app-c-reorderable-list__item" do %>
+      <%= tag.div class: "app-c-reorderable-list__wrapper" do %>
+        <%= tag.div class: "app-c-reorderable-list__content" do %>
+          <%= tag.p item[:title], class: "app-c-reorderable-list__title" %>
+          <%= tag.p item[:description], class: "app-c-reorderable-list__description" %>
+        <% end %>
+        <%= tag.div class: "app-c-reorderable-list__actions" do %>
+          <%= render "govuk_publishing_components/components/input", {
```

Comment:
Side note: is it worth using `<input type=”text” inputmode=”numeric” pattern="[0-9]*">` here as per https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/?

Reply:
Most probably very few/if nay people may benefit from this. Happy to add it if you think it's useful, but will have to update the input component to support these attributes.

Reply:
Opened https://github.com/alphagov/govuk_publishing_components/pull/1345 for this

Reply:
Opened https://github.com/alphagov/govuk_publishing_components/pull/1345 for this